.ci/Jenkinsfile

```diff
@@ -25,6 +25,8 @@ pipeline {
   }
   parameters {
     choice(name: 'LOG_LEVEL', choices: ['INFO', 'DEBUG'], description: 'Log level to be used')
+    choice(name: 'QUERY_MAX_ATTEMPTS', choices: ['5', '10', '20'], description: 'Number of attempts to create the cnnection to Elasticsearch')
```

Comment:
Let's use Jenkins UI to provide different retry strategies for internal retries and timeouts.

If we find it's a timeout problem, because the build agent is overloaded for some reason, then we can use a backoff strategy in the internal retry algorithm, instead of using a fixed (although configurable) one: 5 retries, waiting 3 seconds between each.